Description: Add _FFMPEG to the symbol versions
 This is done to ensure that there is no confusion, if a binary is linked
 against FFmpeg and another shared library, which is linked against Libav.

Author: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Forwarded: not-needed
Last-Update: <2014-09-14>

--- a/libavcodec/libavcodec.v
+++ b/libavcodec/libavcodec.v
@@ -1,4 +1,4 @@
-LIBAVCODEC_$MAJOR {
+LIBAVCODEC_FFMPEG_$MAJOR {
         global: av*;
                 #deprecated, remove after next bump
                 audio_resample;
--- a/libavdevice/libavdevice.v
+++ b/libavdevice/libavdevice.v
@@ -1,4 +1,4 @@
-LIBAVDEVICE_$MAJOR {
+LIBAVDEVICE_FFMPEG_$MAJOR {
         global: avdevice_*; av_*;
         local: *;
 };
--- a/libavfilter/libavfilter.v
+++ b/libavfilter/libavfilter.v
@@ -1,4 +1,4 @@
-LIBAVFILTER_$MAJOR {
+LIBAVFILTER_FFMPEG_$MAJOR {
         global: avfilter_*; av_*;
         local: *;
 };
--- a/libavformat/libavformat.v
+++ b/libavformat/libavformat.v
@@ -1,4 +1,4 @@
-LIBAVFORMAT_$MAJOR {
+LIBAVFORMAT_FFMPEG_$MAJOR {
         global: av*;
                 #FIXME those are for ffserver
                 ff_inet_aton;
--- a/libavresample/libavresample.v
+++ b/libavresample/libavresample.v
@@ -1,4 +1,4 @@
-LIBAVRESAMPLE_$MAJOR {
+LIBAVRESAMPLE_FFMPEG_$MAJOR {
         global: av*;
         local:  *;
 };
--- a/libavutil/libavutil.v
+++ b/libavutil/libavutil.v
@@ -1,4 +1,4 @@
-LIBAVUTIL_$MAJOR {
+LIBAVUTIL_FFMPEG_$MAJOR {
         global: av*;
         local: *;
 };
--- a/libpostproc/libpostproc.v
+++ b/libpostproc/libpostproc.v
@@ -1,4 +1,4 @@
-LIBPOSTPROC_$MAJOR {
+LIBPOSTPROC_FFMPEG_$MAJOR {
         global: postproc_*; pp_*;
         local: *;
 };
--- a/libswresample/libswresample.v
+++ b/libswresample/libswresample.v
@@ -1,4 +1,4 @@
-LIBSWRESAMPLE_$MAJOR {
+LIBSWRESAMPLE_FFMPEG_$MAJOR {
         global: swr_*; swresample_*;
         local: *;
 };
--- a/libswscale/libswscale.v
+++ b/libswscale/libswscale.v
@@ -1,4 +1,4 @@
-LIBSWSCALE_$MAJOR {
+LIBSWSCALE_FFMPEG_$MAJOR {
         global: swscale_*; sws_*;
         local: *;
 };
